### DISCUSSION TITLE
服务器内存与导入的图数据文件大小的关系

### DISCUSSION THREAD

**Discussion by xglreal** [alibaba/GraphScope#2499] [General]
假设我有10G的图数据，需要多少服务器内存才能把该图数据导入？

导入图数据是完全都在内存中，还是也会落在磁盘？

另外，graphscope是否支持gremlin的最短路径算法？

**Comment by sighingnow** [comment]
See also: #2321

**Comment by sighingnow** [comment]
See also https://graphscope.io/docs/frequently_asked_questions.html as well, we have already document the assumption a bit.

**Comment by sighingnow** [comment]
> 导入图数据是完全都在内存中，还是也会落在磁盘？

Yes.

**Comment by sighingnow** [comment]
> 另外，graphscope是否支持gremlin的最短路径算法？

For SSSP, you could try GAE: https://graphscope.io/docs/analytics_engine.html

**Comment by sighingnow** [comment]
Closing as moving to discussion.
